src/backend/storeManagers/zoom/library.ts

```diff
@@ -18,13 +18,16 @@ import {
   logWarning
 } from 'backend/logger'
 import { getFileSize, parseSize } from '../../utils'
+import CacheStore from '../../cache'
 import {
   libraryStore,
   installedGamesStore,
   installInfoStore
 } from './electronStores'
 import { isOnline } from '../../online_monitor'
-import { cachePath, apiUrl } from './constants'
+import { apiUrl } from './constants'
+
+const libraryCache = new CacheStore('zoom-library')
```

Comment:
The `CacheStore` class takes one or two type parameters. In this case, you likely want the following (may not lint):
```suggestion
const libraryCache = new CacheStore<ZoomGameInfo[], 'library'>('zoom-library')
```
With that, you can get rid of the `as ZoomGameInfo[]` when getting the key

Reply:
Done, please take a look.